cms/static/js/spec/views/container_spec.js

```diff
@@ -4,7 +4,7 @@ import EditHelpers from 'js/spec_helpers/edit_helpers';
 import ContainerView from 'js/views/container';
 import XBlockInfo from 'js/models/xblock_info';
 import 'jquery.simulate';
-import 'xmodule/js/src/xmodule';
+import 'xmodule_js/src/xmodule';
```

Comment:
Using the correct symlink name worked here but I wonder how previous one is working on master then?